README.md

```diff
@@ -9,9 +9,23 @@ It provides a new index type for vector columns called `hnsw` which speeds up `O
 
 ## Quickstart
 
-Note: Currently LanternDB depends on [pgvector](https://github.com/pgvector/pgvector) for the `vector` data type. You'll need to manually install pgvector before moving to the next step.
+### Requirements
 
-LanternDB builds and uses [usearch](https://github.com/unum-cloud/usearch) for its single-header state of the art HNSW implementation.
+Make sure you have the following software installed:
+
+- `gcc` (version >= ?)
+- `g++` (version >= ?)
+- `postgres` (version >= ?)
+- `cmake` (version >= ?)
+- `make` (version >= ?)
```

Comment:
These do not go in the main readme. We will have them in our docs. 
However, if you have old versions of each and could help us on which versions LanternDB does not compile, that would be helpful.

Reply:
ohk will remove this for time being